### DISCUSSION TITLE
UI design framework - material?

### DISCUSSION THREAD

**Discussion by klochowicz** [itchysats/10101#20] [Ideas]
In order to make things simple for the tournament, I propose to stick to Material design framework.

link: https://m2.material.io/design/introduction

It's a consistent design language, aimed to be simple to use by non-pros in this space :)
It also feels "familiar" to many people, as it's used by Google everywhere (e.g. GMail). 

With time, we could make the iOS/mac apps look more native by using Cupertino for these platforms (easy to adjust conditionally); but that seems to be out-of-scope for our current stage.

I will start setting up Flutter app with Material (as this is the default). We can always adjust it depending on the outcome of this discussion. 

@da-kami @bonomat @holzeis do you have any strong opinions on this?

**Comment by holzeis** [comment]
I am in favor of this!

**Reply by da-kami** [reply]
I like Material, no objections. It's good to stick to one of the big frameworks for the beginning to have clear guidelines.
I will see to update my knowledge on it and incorporate it into the navigation of the mocks + wireframe.

**Comment by holzeis** [comment]
I am closing this discussion as we are already very much going forward with the material design. Feel free to re-open if you have objections.
